userspace/libsinsp/plugin.cpp

```diff
@@ -328,11 +328,19 @@ bool sinsp_plugin::resolve_dylib_symbols(std::string &errstr)
 			strlcpy(tf.m_display, fdisplay.c_str(), sizeof(tf.m_display));
 			strlcpy(tf.m_description, fdesc.c_str(), sizeof(tf.m_description));
 			tf.m_print_format = PF_DEC;
-
-			if (ftype == "string") {
-				tf.m_type = PT_CHARBUF;
-			} else if (ftype == "uint64") {
-				tf.m_type = PT_UINT64;
+			std::map<std::string, ppm_param_type> pt_lut = {
```

Comment:
```suggestion
			std::unordered_map<std::string, ppm_param_type> pt_lut = {
```
NIT: This should give us O(1). Not really relevant here, but better setting solid ground already. Moreover, this map could potentially be a const static field of the `sinsp_plugin` class, given that it never changes.